notebooks/00_scratch_r.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+title: "00_scratch_r"
+output:
+#  html_document: default
+  pdf_document: default
+#  md_document:
+#    variant: markdown_github
+
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+```{r, echo=FALSE}
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+```
+
+## Global defaults
+
+```{r}
+options(repr.plot.width = 6, repr.plot.height = 6)
+
+```
+
+
+## Utility functions
+
+```{r}
+# TODO refactor this 
+print_ln <- function() {
+  cat(  "--------------------------------------------------------------------------------\n"  )
+}
+```
+
+
+```{r}
+life_expectancy <- read.csv("../data/raw/UNdata_Export_20200411_102825637.csv", stringsAsFactors = FALSE)
```

Comment:
What is the use of `stringsAsFactors`

Reply:
Actually, I found a good exercise on DataCamp which explains this better 
https://campus.datacamp.com/courses/introduction-to-r-for-finance/factors-4?ex=10